.gitignore

```diff
@@ -0,0 +1,4 @@
+*.csv
```

Comment:
These lines will make git ignore files. This is helpful when you have your datasets in the same folder as your code. In my case, I have the csv's and the zips in there.